rental.rb

```diff
@@ -1,3 +1,5 @@
+require 'json'
+
 class Rental
   attr_accessor :date, :book, :person
 
```

Comment:
- [ ] The person data is incorrect, please use the documentation on how to attribute Id right now is giving error.

![Capture d’écran 2022-01-14 155642](https://user-images.githubusercontent.com/61562626/149536162-0d307330-5158-464b-aeb3-614e2ccb5c8b.png)

Reply:
@Dipeshtwis Hello 
im sure that you are try on diffrent repo becuase this working fine 
![image](https://user-images.githubusercontent.com/88675267/149540792-fa1308b4-e57d-49cb-a7e3-92b0a3c6a270.png)

please re check again because its not my fault

Reply:
The repo is your's, and the data from rental is causing problem in  person.rb